packages/extension-ui/src/components/themes.ts

```diff
@@ -2,34 +2,38 @@
 // SPDX-License-Identifier: Apache-2.0
 
 const darkTheme = {
-  accountDotsIconColor: '#8E8E8E',
+  accountDotsIconColor: '#00CCAB',
   addAccountImageBackground: '#1A1B20',
+  avatarBorderColor: '#FFFFFF',
   backButtonBackground: '#3A3B41',
   backButtonBackgroundHover: '#3a3b41ad',
   backButtonTextColor: '#FFFFFF',
-  background: '#26272C',
+  background: 'radial-gradient(104.44% 39.97% at 50% 0%, #14202A 41.83%, rgba(20, 32, 42, 0) 100%), #111B24;',
   bodyColor: '#20222A',
-  borderRadius: '4px',
-  boxBackground: '#1A1B20',
-  boxBorderColor: '#303030',
+  borderRadius: '8px',
+  boxBackground: '#111B24',
+  boxBorderColor: '#1B2B38',
   boxLineHeight: '1rem',
   boxMargin: '0.75rem 0',
   boxPadding: '0 0.25rem',
   boxShadow: 'rgba(0, 0, 0, 0.86)',
-  buttonBackground: '#E86F00',
+  buttonBackground: '#00EAC7',
   buttonBackgroundDanger: '#AF1111',
   buttonBackgroundDangerHover: '#D93B3B',
-  buttonBackgroundHover: '#ED9329',
-  buttonTextColor: '#FFFFFF',
+  buttonBackgroundHover: '#7ff4e3',
+  buttonBorderRadius: '24px',
+  buttonTextColor: '#111B24',
   connectedDotColor: 'seagreen',
   errorBorderColor: '#7E3530',
   errorColor: '#E42F2F',
-  fontFamily: 'Nunito, sans-serif',
+  fontFamily: 'Karla, sans-serif',
+  fontFamilyGilroy: 'Gilroy, sans-serif',
+  fontFamilyKarla: 'Karla, sans-serif',
```

Comment:
Can we maybe choose the one with most uses and call it `primaryFontFamily` and the second one `secondaryFontFamily`?  Yes it would probably require change the use cases of `fontFamily` in a few places but we're rebranding whole app and having three entries here look a bit confusing 😅

Reply:
fixed in [9898c8f](https://github.com/Cardinal-Cryptography/aleph-wallet-extension/pull/1/commits/9898c8f481eb186e26f6b1e93759358de8be7b8d) && [56c3f14](https://github.com/Cardinal-Cryptography/aleph-wallet-extension/pull/1/commits/56c3f144a0db7e0927cdddf2a9b1390e4a4e581f)